specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/policyStates.json

```diff
@@ -0,0 +1,1737 @@
+          "Query latest at management group scope": {
+            "$ref": "./examples/PolicyStates_QueryManagementGroupScope.json"
+          }
+        }
+      }
+    },
+    "/providers/{managementGroupsNamespace}/managementGroups/{managementGroupName}/providers/Microsoft.PolicyInsights/policyStates/{policyStatesSummaryResource}/summarize": {
+      "post": {
+        "operationId": "PolicyStates_SummarizeForManagementGroup",
+        "description": "Summarizes policy states for the resources under the management group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/policyStatesSummaryResourceParameter"
+          },
+          {
+            "$ref": "#/parameters/managementGroupsNamespaceParameter"
+          },
+          {
+            "$ref": "#/parameters/managementGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/topParameter"
+          },
+          {
+            "$ref": "#/parameters/fromParameter"
+          },
+          {
+            "$ref": "#/parameters/toParameter"
+          },
+          {
+            "$ref": "#/parameters/filterParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Summarize results.",
+            "schema": {
+              "$ref": "#/definitions/SummarizeResults"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/QueryFailure"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Summarize at management group scope": {
+            "$ref": "./examples/PolicyStates_SummarizeManagementGroupScope.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.PolicyInsights/policyStates/{policyStatesResource}/queryResults": {
+      "post": {
+        "operationId": "PolicyStates_ListQueryResultsForSubscription",
+        "description": "Queries policy states for the resources under the subscription.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/policyStatesResourceParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/topParameter"
+          },
+          {
+            "$ref": "#/parameters/orderByParameter"
+          },
+          {
+            "$ref": "#/parameters/selectParameter"
+          },
+          {
+            "$ref": "#/parameters/fromParameter"
+          },
+          {
+            "$ref": "#/parameters/toParameter"
+          },
+          {
+            "$ref": "#/parameters/filterParameter"
+          },
+          {
+            "$ref": "#/parameters/applyParameter"
```

Comment:
what about expand for this

Reply:
There's a separate path with expandParameter at resourceId scope that we currently support. I believe it's not supported at other scopes.

---
In reply to: [335637526](https://github.com/Azure/azure-rest-api-specs/pull/7505#discussion_r335637526) [](ancestors = 335637526)